homeassistant/components/linkplay/services.yaml

```diff
@@ -13,3 +14,18 @@ play_preset:
           min: 1
           max: 10
           mode: box
+
+send_command:
+  name: HTTP API Command
+  description: Execute HTTP API linkplay-specific commands on the player. Be careful! Some commands can cause damage to your device if you don't know what you're doing.
```

Comment:
Let's add the name and description to `strings.json`. That way they can be translated into other languages.